source/develop/release-management/features/engine/metrics-store.html.md

```diff
@@ -23,20 +23,79 @@ Real-time monitoring will provide visibility into the user's complete infrastruc
 
 *   Target Release: 4.1
 *   Status: Released
-*   Last updated: Mon 24 Jan 2017
+*   Last updated: Mon 12 Jun 2017
 
 ## Description
 
-[![oVirt Metrics Store Architecture](/images/wiki/MetricsStoreArchitecture.jpg)](images/wiki/MetricsStoreArchitecture.jpg)
+In 4.1, Collectd and Fluentd are already installed on the hosts and engine machines.
```

Comment:
collectd should be all lowercase (http://collectd.org), fluentd as well, all lowercase (http://www.fluentd.org/)